model/project_ref.go

```diff
@@ -404,6 +410,49 @@ func (p *ProjectRef) GetPatchTriggerAlias(aliasName string) (patch.PatchTriggerD
 	return patch.PatchTriggerDefinition{}, false
 }
 
+func (p *ProjectRef) GetMergedConfig(version string) *MergedProjectConfig {
+	lookupVersion := false
+	if version == "" {
+		lastGoodVersion, err := FindVersionByLastKnownGoodConfig(p.Id, -1)
+		if err != nil || lastGoodVersion == nil {
+			return nil
+		}
+		version = lastGoodVersion.Id
+		lookupVersion = true
+	}
+	parserProject, err := ParserProjectFindOneById(version)
+	if err != nil {
```

Comment:
I don't know that we should be suppressing these errors.

Reply:
What I meant by this is that we should return the error; an error here is a database error, not a "this project doesn't exist" instance so we should error on it and the user or job or etc can try again.